_docs/_latest/setup/manual.md

```diff
@@ -51,7 +50,7 @@ Where `#######` is a random alphanumeric unique identifier.
 
 For information about the roles you need to assign to the Forseti server
 service account, see 
-[the server service account]({% link _docs/latest/concepts/service-accounts.md %}#the-server-service-account).
+[the server service account]({% link _docs/latest/concepts/service-accounts.md#the-server-service-account%}).
```

Comment:
Does this work? I think the previous way was how to link to a particular heading while also using the {% link } method.

Reply:
Ah, I might have gotten it wrong, I haven't been able to get staging to work. If you have time to help troubleshoot that, maybe we can sync up? I can just revert this change though, if that's best?

Reply:
Resolved.